autocomplete/definitions/global/table/binsearch.lua

```diff
@@ -1,19 +1,24 @@
 return {
 	type = "function",
-	description = [[Performs a binary search for a given value.
+	description = [[Performs a binary search for a given `value` inside a specified `table` `t`.
 
-If the value is found: It returns a table holding all the mathing indices (e.g. { startindice,endindice } ) endindice may be the same as startindice if only one matching indice was found
+If the `value` is found in `t`, then a `table` providing the range of all matching indices is returned. (e.g. `{ startindice, endindice }`.) 
+If only one matching index was found, then `startindice` will be the same as `endindice`.
 
-If compval is given: then it must be a function that takes one value and returns a second value2, to be compared with the input value, e.g.: compvalue = function( value ) return value[1] end
+If `value` is not found in `t`, then `nil` is returned.
 
-If reversed is set to true: then the search assumes that the table is sorted in reverse order (largest value at position 1) note when reversed is given compval must be given as well, it can be nil/_ in this case
+If `compval` is given, then it must be a function that takes one value and returns the value to use for comparisons. For example, to compare arrays based on their first entry, you can write `compvalue = function( value ) return value[1] end`.
```

Comment:
I see that this is as it was before, but let's  now update it to be consistent with the rest of the project `compvalue = function(value) return value[1] end`